.travis.yml

```diff
@@ -14,8 +14,9 @@ install:
   - sudo apt-get install docker-engine
   - docker swarm init --advertise-addr 127.0.0.1
   - make install
```

Comment:
Yeah, we could replace the commands with
`docker run -it --rm -v /var/run/docker.sock appcelerator/amp:local amp platform pull --local`
`docker run -it --rm -v /var/run/docker.sock appcelerator/amp:local amp platform start --local`
`docker run -it --rm -v /var/run/docker.sock appcelerator/amp:build make test`